static/js/stream_list.js

```diff
@@ -21,6 +22,11 @@ function active_stream_name() {
 
 function filter_streams_by_search(streams) {
     var search_box = $(".stream-list-filter");
+    if (search_box.length === 0) {
+        // If no stream filter box is there,
+        // return all streams.
+        return streams;
+    }
```

Comment:
If this is just for the Node tests, yeah, I'd just create this in the Node test code...